alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/SagaActor.java

```diff
@@ -55,7 +59,7 @@
 
 public class SagaActor extends
     AbstractPersistentFSM<SagaActorState, SagaData, DomainEvent> {
-
+  protected static final int PAYLOADS_MAX_LENGTH = 10240;
```

Comment:
We need to be able to configure this parameter from setting environment variable.

Reply:
Agree, I found static variable PAYLOADS_MAX_LENGTH in class TxEvent, I'll solve this problem in another PR

Reply:
I created JIRA https://issues.apache.org/jira/browse/SCB-1811 for it